### PR TITLE
Move a function in order to remove a circular import

### PR DESCRIPTION
**Pull request by Pierre-Sassoulas**
## Description

Fix a TODO and a suspicious code construct, one can wonder why it was this way. But the tests seems to still work.

## Type of Changes

|     | Type                   |
| --- | ---------------------- |
| ✓   | :hammer: Refactoring   |

### CODE REVIEW

**Comment by Pierre-Sassoulas** [comment]
@cdce8p could you review this one please ? :)

File: astroid/bases.py [hunk 1]
```diff
@@ -283,6 +283,23 @@ def callable(self):
         except exceptions.AttributeInferenceError:
             return False
 
+    def getitem(self, index, context=None):
```
Comment (cdce8p) [inline astroid/bases.py:286]: Why not put it at its original position (L319)?

**Review by cdce8p — APPROVED** [review]
One small comment. Besides that, the change looks fine. Although I also can't really tell why it was done that way to begin with and thus am not 100% sure it's safe to do. Anyway pylint tests pass, so I guess the circular dependency isn't present anymore.

File: astroid/bases.py (hunk 1 above)
Comment (Pierre-Sassoulas) [reply astroid/bases.py]: Just realized the function already existed after moving it :) Fixed.

**Pull request merged**